ext_tables.sql

```diff
@@ -59,7 +68,7 @@ CREATE TABLE tx_t3blog_com_nl (
     name tinytext NOT NULL,
     post_uid int(11) DEFAULT '0' NOT NULL,
     lastsent int(11) DEFAULT '0' NOT NULL,
-    code tinytext NOT NULL,
+    code varchar(20) DEFAULT '' NOT NULL,
```

Comment:
Are you sure 20 chars are enough? We create the code with a 32 char max, see https://github.com/fnagel/t3extblog/blob/master/Classes/Domain/Model/AbstractSubscriber.php#L155

Reply:
I didn't see the max size of the char in AbstractSubscriber.php
I increased the value to 32